test/token/ERC20/extensions/ERC20Burnable.behavior.js

```diff
@@ -1,112 +1,91 @@
-const { BN, constants, expectEvent } = require('@openzeppelin/test-helpers');
-const { ZERO_ADDRESS } = constants;
-
+const { ethers } = require('hardhat');
 const { expect } = require('chai');
-const { expectRevertCustomError } = require('../../../helpers/customError');
 
-function shouldBehaveLikeERC20Burnable(owner, initialBalance, [burner]) {
+function shouldBehaveLikeERC20Burnable() {
```

Comment:
I think this behavior is very specific to ERC20, is there a reason to keep it like this instead of inline it in ERC20Burnable @Amxx?

Same for ERC20Capped.behavior

Reply:
I'd be ok with inlining them.

Reply:
> I'd be ok with inlining them.

Let's do it 👍🏻